### PR TITLE
Update Error Handling doc to latest, add Turkish translation

### PR DESCRIPTION
**Pull request by ahmetomerv**

### CODE REVIEW

File: tr/guide/error-handling.md [hunk 1]
```diff
@@ -1,24 +1,170 @@
...
 ---
-<div id="page-doc" markdown="1">
-# Error handling
+# Hata İşleme
 
-Define error-handling middleware functions in the same way as other middleware functions,
-except error-handling functions have four arguments instead of three:
-`(err, req, res, next)`. For example:
+
+_Error Handling_ refers to how Express catches and processes errors that
```
Comment (crandmck) [inline tr/guide/error-handling.md]: At first, I thought this was an untranslated paragraph, but I now realize that it's just the original English and the Turkish follows.   In general, for localized content, the process we've followed is to REPLACE the en content, not duplicate & translate.  

Unless there is a specific good reason, please remove the en content from the translated page.

File: tr/guide/error-handling.md [hunk 2]
```diff
@@ -1,24 +1,170 @@
...
+  var user = await getUserById(req.params.id)
+  res.send(user)
+})
+```
+
+`getUserById` bir hata fırlattığında veya ret verdiğinde, `next` fonksiyonu ya fırlatılan hatayla ya da ret verilen değer ile çağrılacaktır. Eğer herhangi bir ret değeri verilmediyse, `next` fonksiyonu Express yönlendiricisi tarafından sağlanan varsayılan Error objesiyle çağrılacak.
+
+Eğer `next()` fonksiyonuna herhangi birşey verdiğinizde (`'route'` karakter dizisi hariç), Express şimdiki isteği bir hata olarak sayıp hata işlemeyen yönlendirici ve ara yazılım fonksiyonlarını es geçecektir.
+
+If the callback in a sequence provides no data, only errors, you can simplify
```
Comment (crandmck) [inline tr/guide/error-handling.md]: Another en para.

File: tr/guide/error-handling.md [hunk 3]
```diff
@@ -65,21 +207,22 @@ function logErrors (err, req, res, next) {
...
-    res.status(500).send({ error: 'Something failed!' })
+    res.status(500).send({ error: 'Birşeyler ters gitti' })
   } else {
     next(err)
   }
 }
 ```
 
-The "catch-all" `errorHandler` function might be implemented as follows:
+Implement the "catch-all" `errorHandler` function as follows (for example):
```
Comment (crandmck) [inline tr/guide/error-handling.md]: Relict en para :-)

**Review by crandmck — CHANGES_REQUESTED** [review]

File: tr/guide/error-handling.md (hunk 1 above)
Comment (ahmetomerv) [reply tr/guide/error-handling.md]: My mistake, I was supposed to remove the English version once the translation was done. Removed them all, thanks for reviewing.

**Review by crandmck — APPROVED** [review]

**Pull request merged**